public/js/utils/databus-webapp-utils.js

```diff
@@ -18,6 +18,9 @@ class DatabusWebappUtils {
   }
 
   logout() {
+    //to actually remove the saved session 
+    localStorage.clear();
```

Comment:
I am worried that this would also delete all drafts in the collection editor (that are stored in local storage). in general that sounds like data loss to me (just clearing it).

Reply:
I thought it would clear all the cache of the website with the user session and once the user log in, they get all drafts by the server.

Reply:
or i should close the pr

Reply:
well I am not sure about this behaviour. are you guessing or did you have a look into code and try it out.
on my side I can confirm that calling localStorage.clear() in the console after I just created a new collection draft
this is lost (both scenarios if I only let the draft open in the window without clicking any buttons or links, but also when I go back to the overview of all collections which shows me that draft in the list)

Reply:
@JJ-Author makes much sense